.github/workflows/hotfix-wf-service.yml

```
@@ -0,0 +1,400 @@
+      - name: Bump version
+        id: bump-version
+        uses: ./.github/actions/bump-version
+        with:
+          tag: ${{ steps.version.outputs.tag }}
+      
+      - name: "Determine Branch"
+        id: sanitized-branch
+        uses: transferwise/sanitize-branch-name@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)